packages/react-components/src/templates/__tests__/ProfileHeader.test.tsx

```diff
@@ -68,11 +68,6 @@ it('does not show team information if the user is not on a team', async () => {
 });
 
 it('generates the last updated text', () => {
-  const { container } = render(
-    <ProfileHeader
-      {...boilerplateProps}
-      orcidLastModifiedDate={formatISO(subYears(new Date(), 2))}
-    />,
-  );
+  const { container } = render(<ProfileHeader {...boilerplateProps} />);
```

Comment:
sorry to be a pain but can I ask you to revert moving the prop to the `boilerplateProps`? The `expect`ation is confusing if you can't see straight in the test why it should be "2 years ago". All the other tests pass those props that are relevant to them themselves.

Reply:
Ok, I can override it. `lastModifiedDate` is required now.